app/views/reviews/_review_content.html.erb

```diff
@@ -0,0 +1,97 @@
+<% show_full_review = false unless local_assigns[:show_full_review].present? %>
```

Comment:
This logic lets us have the same partial for showing the full review and for showing the summary of the review. Useful for now, but might change later. The only difference between the two atm is that the full review.comment is always shown if show_full_review is true.

Reply:
I would prefer that show_full_review is sent by whoever renders this, it's usually good practice to not do assignments in view files. I think it makes for now as it only changes the review.comments. But if in the future the difference between full and not full review gets larger they probably should be split up into partials

Reply:
Thinking here was to not HAVE to send show_full_review every time I call the partial, by setting it to false if it's not set, so we can test for it in the view file regardless of if it's called or not. 

Not sure if it's a good way to do it at all though...

Your recommendation is we keep it as it is, or change it?

Reply:
If you don't pass show_full_review to the partial it will become undefined which means
`if show_full_review` will be false. So I think the best is to change it

Reply:
Hm maybe keep it as is and just change it to
```
<% show_full_review = false unless defined?(:show_full_review) %>
```

Reply:
Fixed!